06-statistical-testing.Rmd

```diff
@@ -523,10 +525,10 @@ chi_ex2$observed %>% as_tibble() %>%
   gt(rowname_col = "TrustGovernment") %>%
   tab_stubhead(label = "Trust in Government") %>%
   tab_spanner(label = "Trust in People",
-              columns = everything())
+              columns = everything()) 
```

Comment:
Table goes off page.  Need to adjust.  Think can just adjust the column headers but will need to play with it a bit.